Resources/Prototypes/Entities/Mobs/NPCs/miscellaneous.yml

```
@@ -79,6 +79,53 @@
   name: tomato killer
   description: Looks like it's not you eating tomatoes today, it's the tomatoes eating you.
   components:
+  - type: GhostRole
+    prob: 0.15
```

```suggestion
    prob: 0.05
```
Should be a little lower considering how many tomatos can be harvested in a short amount of time. We don't want to spam ghost roles.